package.json

```diff
@@ -12,7 +12,8 @@
         "format-check": "prettier --check \"src/**/*.{ts,tsx}\" \"test/**/*.{ts,tsx}\"",
         "format-overwrite": "prettier -w \"src/**/*.{ts,tsx}\" \"test/**/*.{ts,tsx}\"",
         "test": "cross-env TS_NODE_FILES=true mocha -r ts-node/register \"./test/**/*.spec.ts\"",
-        "coverage": "nyc --cache npm test && nyc report --reporter=html && echo HTML report generated at coverage/index.html",
+        "test-all": "cross-env TS_NODE_FILES=true mocha --forbid-only -r ts-node/register \"./test/**/*.spec.ts\"",
```

Comment:
Should be rather make this default "test" script instead of another "test-all" ?

Reply:
I didn't want to stomp on 'test' in case folks are using that as their main test runner .. e.g. 
```
mark something with .only
$ npm run test
```

Reply:
Hmmm true, let's keep both for now and based on developers request/feedback we can change it later.